Resources/config/jackalope_doctrine_dbal.xml

```diff
@@ -13,15 +13,15 @@
 
         <service id="doctrine_phpcr.jackalope_doctrine_dbal.schema"
                  class="%doctrine_phpcr.jackalope_doctrine_dbal.repository_schema.class%"
-                 public="false"
+                 public="true"
         >
             <argument type="collection"/>
             <argument type="service" id="doctrine_phpcr.jackalope_doctrine_dbal.default_connection"/>
         </service>
 
         <service id="doctrine_phpcr.jackalope_doctrine_dbal.schema_listener"
                  class="%doctrine_phpcr.jackalope_doctrine_dbal.schema_listener.class%">
-            <argument type="service" id="doctrine_phpcr.jackalope_doctrine_dbal.schema" />
+            <argument type="service" id="service_container" />
```

Comment:
Would also prefer doing a `lazy` true as this could be a BC Break if somebody did overwrite the service

Reply:
As discussed here: https://github.com/doctrine/DoctrinePHPCRBundle/pull/339#discussion_r351168745